### PR TITLE
Remove unnecessary else statement. Simplify it.

### PR DESCRIPTION
**Pull request by irazasyed**

### CODE REVIEW

**Comment by irazasyed** [comment]
I wonder why there's an issue with StyleCI when i applied CS Fixer and it isn't able to find any issue?

File: src/Illuminate/Foundation/helpers.php [hunk 1]
```diff
@@ -102,9 +102,9 @@ function auth($guard = null)
     {
         if (is_null($guard)) {
             return app(AuthFactory::class);
-        } else {
-            return app(AuthFactory::class)->guard($guard);
         }
+
```
Comment (rkgrep) [inline src/Illuminate/Foundation/helpers.php]: You have unnecessary spaces here.

Comment (irazasyed) [inline src/Illuminate/Foundation/helpers.php]: Weird. Done, removed and committed, seems to be working :+1:

**Comment by GrahamCampbell** [comment]
Sorry, we don't accept PRs of this nature.

**Comment by irazasyed** [comment]
May i suggest the contributing guidelines list the nature of PRs you guys accept? You know it'll save time for developers as well as you from reviewing and then closing when its not going to be accepted.

**Pull request closed without merging**